### ISSUE TITLE
[DOCS]: Multiple suggestions

### ISSUE THREAD

**Issue by cyber-sec0** [AIHawk-FOSS/Auto_Jobs_Applier_AI_Agent#988]
### Affected documentation section

Affects multiple parts, each of my comments refers to a different documentation improvement.

### Documentation improvement description

Does the program support multiple experiences and education details? (If not it should be added as a feature)
Is it also possible to erase exam_name_x? (If I don't want to include it) Or instead of erasing, should I leave it as it is "exam_name_1: "[Grade]"" and so on?

Can achievements be completely erased?

Can salary_range_usd be an hourly rate?

This wasn't clear on the documentation so I had to ask...
It should be added whether or not this is possible.

### Why is this change necessary?

Clarification

**Comment by cyber-sec0** [comment]
The file guide_to_setup_ollama_and_gemini.pdf says that Gemini won't work unless you add your card.
This is incorrect because they do have free tiers, as I've made an AI app that uses Gemini before and I never had to pay for it.

**Comment by cyber-sec0** [comment]
The .pdf files on the documentation where deleted and now redirect to 404 github pages.

**Comment by cyber-sec0** [comment]
pip install lib_resume_builder_AIHawk doesn't work unless https://visualstudio.microsoft.com/visual-cpp-build-tools/ is installed.
This isn't mentioned anywhere on the documentation.

**Comment by cyber-sec0** [comment]
if both parameters below still exist, add them to the documentation.

python main.py --resume ~/Downloads/Resume.pdf

python main.py --collect #collect job data only to perform any type of data analytics you can use the bot with the --collect option. This will store in output/data.json file all data found from linkedin jobs offers.

**Comment by cyber-sec0** [comment]
I tried to install the requirements.txt as mentioned on the documentation, but it never worked so I had to manually do it one by one!

Perhaps erasing the requirements.txt and leaving all lines below on the documentation would be better?

```
pip install click
pip install inquirer
pip install pyyaml
pip install selenium
pip install webdriver-manager
pip install openai
pip install langchain_core
pip install langchain_openai
pip install loguru
pip install langchain_text_splitters
pip install langchain_community
pip install lib_resume_builder_AIHawk
pip install pydantic[email]
```

**Comment by cyber-sec0** [comment]
I saw on your video that you added email/password on the config file, but the docs don't say that and the current config file doesn't come with it, so I guess that's no longer needed?
Or is it not working because I haven't added it?

I also removed the quotes from my API key, but that didn't change anything either

email: eee
password: eee
llm_api_key: eee

Both of these questions should be answered in the documentation.
